app/pages/Home/index.ts

```diff
@@ -112,33 +22,39 @@ export default class Home extends Page implements SitePage {
 
   createShowcase() {
     this.projectShowcase = new ProjectShowcase({
-      title: PROJECTS_MOCK[this.projectId]?.title,
-      videoUrl: PROJECTS_MOCK[this.projectId]?.videoUrl,
-      description: PROJECTS_MOCK[this.projectId]?.description,
+      title: PROJECTS[this.projectId]?.title,
+      videoUrl: PROJECTS[this.projectId]?.videoUrl,
+      description: PROJECTS[this.projectId]?.description,
       number: `0${this.projectId + 1}.`,
-      url: PROJECTS_MOCK[this.projectId]?.project_links?.url,
-      github_url: PROJECTS_MOCK[this.projectId]?.project_links?.github_url,
+      url: PROJECTS[this.projectId]?.project_links?.url,
+      github_url: PROJECTS[this.projectId]?.project_links?.github_url,
+      frontendTech: PROJECTS[this.projectId]?.technologies.frontend,
+      backendTech: PROJECTS[this.projectId]?.technologies.backend,
     })
   }
 
   createMedia() {
-    PROJECTS_MOCK.map((project, i) => {
+    PROJECTS.map((_, i) => {
       const videoClips: NodeListOf<HTMLVideoElement> =
         document.querySelectorAll(`.project__card__video__clip`)
 
       if (videoClips && videoClips.length) {
-        videoClips[i].src = PROJECTS_MOCK[i].videoUrl
-        videoClips[i].load()
+        videoClips[i].src = PROJECTS[i].videoUrl
+        // videoClips[i].load()
```

Comment:
Clean up the videos and refactor so it's obvious this is all static imagery. Can add links to video content so it's optional e.g.(on YouTube) since people are busy